packages/devtools_app/lib/src/shared/console/console_service.dart

```diff
@@ -74,6 +74,34 @@ class VariableConsoleLine extends ConsoleLine {
 /// Source of truth for the state of the Console including both events from the
 /// VM and events emitted from other UI.
 class ConsoleService extends Disposer {
+  void appendBrowsableInstance({
+    required InstanceRef? instanceRef,
+    required IsolateRef? isolateRef,
```

Comment:
When will we have a null isolate?

Reply:
It is nullable for consistency with other methods in this class. May be we should refactor/review the code to deal better with nullability of IsolateRef.